worth2/main/generic/models.py

```diff
@@ -57,6 +54,12 @@ def unlocked(self, user):
 
         return True
 
+    def pageblock(self):
+        return self.pageblocks.first()
+
+    # TODO: I'd like to have all the following methods be inherited
```

Comment:
Interesting.. I did some reading about ABCs, and tried out a few things. It's a little confusing imagining how this would fit in with Django's model-specific `abstract = True` mechanism. I'm looking forward to finding some way to automate the form creation for custom pageblocks, but haven't managed to do that yet by setting `__metaclass__ = ABCMeta`.